roles/reportinator_server/templates/docker-compose.yml.tpl

```diff
@@ -0,0 +1,23 @@
+---
+version: "3.3"
+
+services:
+  reportinator_server:
+    image: "{{ reportinator_server_image }}:{{ reportinator_server_image_tag }}"
+    container_name: "reportinator_server"
+    restart: always
+    volumes:
+      - data:/app/data
+    labels:
+      - "traefik.enable=true"
```

Comment:
Maybe I'm missing it but I don't see the `reportinator_server` role depending on the `traefik` role anywhere. Maybe we don't need traefik at all for this app at this time? In which case we could remove these labels. 

Also do we need an SSL cert? If so we might need `certbot_cloudflare` too.

Reply:
it's using traefik, but now the traefik service is coming from the traefik role and it connects to this docker-compose through the network `proxy`: https://github.com/planetary-social/ansible-scripts/blob/main/roles/traefik/templates/docker-compose.yml.tpl